.github/pull_request_template.md

```diff
@@ -1,3 +1,7 @@
+<!-- PR guidance notes (you can delete this section or leave as-is after reading)
+If your PR is still being drafted you should submit it as a "Draft Pull Request" or add [WIP] to your title.
+Remember if you are a contributor you can set the PR type from Labels if one fits (prefix PR:), and associate your PR with the current "Project". If you can't see this stuff we'll set it up for you. -->
```

Comment:
Can you drop the `If you can't see this stuff` and possibly clarify who a contributor is. I think you mean a member of CorsixTH, which is a pretty small group of people - maybe drop that whole section?